updates/1.0164/161224_create_table_vacationnotification.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The folder of this update should be 1.104 and the timestamp in the filename probably more like 210202 (usually useful when trying to figure out when something was introduced).